packages/block-library/src/image/style.scss

```diff
@@ -1,7 +1,5 @@
 .wp-block-image {
-	// The image block is in a `figure` element, and many themes zero this out.
-	// This rule explicitly sets it, to ensure at least some bottom-margin in the flow.
-	margin-bottom: 1em;
```

Comment:
The image used to have this opinionated margin and I'm not sure why it's there specially since it's not added to the other blocks, I believe being consistent between blocks is better. cc @jasmussen

Reply:
I vaguely recall this being related to image captions. But any spacing issues there could presumably be handled by the figcaption, for which we have these rules:

```
.wp-block-image figcaption {
    margin-top: 0.5em;
    margin-bottom: 1em;
}
```